vmware-event-router/Dockerfile

```diff
@@ -18,6 +18,8 @@ COPY internal internal
 RUN ./bin/golangci-lint run ./...
 RUN CGO_ENABLED=0 GOOS=linux go build -a -installsuffix nocgo -ldflags="-X main.version=${VERSION} -X main.commit=${COMMIT}" -o vmware-event-router cmd/main.go
 
+RUN go test $(go list ./...) -cover
```

Comment:
can we move "cover" into the `Makefile` and `test` target please?

Reply:
Done :heavy_check_mark: